### ISSUE TITLE
install problem

### ISSUE THREAD

**Issue by uloveme99** [Cacti/plugin_gexport#34]
When i wanted to install plugins, I found plugin of Gexport shows “Plugin directory is not correct. Should be ‘export’ but is ‘gexport’.”
This is my screen shot:
![2019-02-13_153908-1](https://user-images.githubusercontent.com/30061894/52827589-6b366a00-3100-11e9-989e-9751331d5c7f.jpg)
I try to copy directory of gexport to a new directory of export.
But still i can’t install plugin of Gexport.
Would you give me some advises to resolve it, thanks!

**Comment by netniV** [comment]
It would appear your screenshot isn't working.

**Comment by uloveme99** [comment]
> It would appear your screenshot isn't working. If you have the image copied into memory, you can paste directly into the reply box and it will automatically upload/link to it for you.
Hi,netniV:
Thanks for  your help!!

**Comment by netniV** [comment]
OK, so the answer is in the screenshot.  Rename the directory to the correct name.

**Comment by cominderace** [comment]
I have similar problem: 
1) according to the plugin docs - directory name should ne gexport
2) however in Plugin Management it is suggested "directory name should be export"
3) after renaming directory to export - it is listed in All or Not Installed plugings list
4) when we move cursor over Actions icon - "Install Plugin" item appears as tooltip, but neither possible to click inside tooltip not clicking on Actions icon have any effect.
In other words - we can not install and activate it.

**Comment by cominderace** [comment]
Our cacti version: 1.2.7. on CentOS 7.  LAPM with PHP7.

**Comment by cominderace** [comment]
![image](https://user-images.githubusercontent.com/56590460/74610553-e7410480-50fc-11ea-9e36-e30da53315bb.png)
when dircetory name ir gexport - we see 2 items listed (Export and Gexport). 
With Gexport we see Plugin description, Author and Version (1.2).
So, something is wrong.

**Comment by cominderace** [comment]
Pluging (ZIP) has been downloaded today from https://github.com/Cacti/plugin_gexport/tree/master

**Comment by netniV** [comment]
Ah, this is simply a case of using the wrong branch.  You should either use the Releases or Develop branch.  But I will get the master branch updated to the latest release version now.
